src/wovnio/wovnphp/Url.php

```diff
@@ -238,6 +251,28 @@ public static function shouldIgnoreBySitePrefixPath($uri, $settings)
         return false;
     }
 
+    public static function parseUrlStr($url)
```

Comment:
Why aren't you using `parse_url` here?

Reply:
I want to use parse_url, but parse_url is not enough.
When parse `testsite.com` with parse_url, path will be `testsite.com`.
So, I needed to wrap parse_url.

Reply:
I'm not sure what you mean.
```
print_r(parse_url("http://test.com/dir/page.html"));

Array
(
    [scheme] => http
    [host] => test.com
    [path] => /dir/page.html
)
```

Reply:
I mean that `json_encode(parse_url('testsite.com')); ` will be `{"path":"testsite.com"}`.

Reply:
When are you encountering URLs that do not have the protocol? We should be doing additional work in wovn.php to ensure the URL is in a consistent format.

Reply:
If we do not have access to the full URL, how do we expect to implement custom domain properly? We need to know if the incoming request is coming from "ja.site.com/foo" vs "en.site.com"

Reply:
This should be similar to the subdomain pattern where it uses HTTP_HOST. I understand HTTP_HOST may not always be available depending on configuration, in that case we should fallback to the default custom domain host.